common/datasets/tedlium2/corpus.py

```diff
@@ -0,0 +1,82 @@
+import os
+from functools import lru_cache
+from typing import Dict, Optional
+
+from sisyphus import tk
+
+from i6_core.audio.encoding import BlissChangeEncodingJob
+
+from i6_core.meta import CorpusObject
+
+from .constants import DURATIONS
+from .download import download_data_dict
+
+
+@lru_cache()
+def get_bliss_corpus_dict(
+    audio_format: str = "wav", output_prefix: str = "datasets"
+) -> Dict[str, tk.Path]:
+    assert audio_format in ["flac", "ogg", "wav", "sph", "nist"]
```

Comment:
please clearly indicate that `sph` and `nist` mean the same thing. Is there even a reason why you need both?